fastlane/lib/fastlane/actions/puts.rb

```diff
@@ -13,6 +13,17 @@ def self.description
         "Prints out the given text"
       end
 
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :message,
```

Comment:
I think in ` def self.run(params)` you're going to want to inspect `params` for `params[:message]`
if it exists, basically, something like:
```
unless params[:message].nil?
  UI.message(params[:message]) 
  return
end
 UI.message(params.join(' '))

I think this would fix the test errors too